modules/tinymce/src/models/dom/test/ts/browser/table/command/TableDeleteColumnTest.ts

```diff
@@ -141,10 +141,10 @@ describe('browser.tinymce.models.dom.table.command.TableDeleteColumnTest', () =>
         TinySelections.setCursor(editor, [ ...pathToBody, 1, 0, 0 ], 0);
         editor.execCommand('mceTableDeleteCol');
         TinyAssertions.assertCursor(editor, [ ...pathToBody, 1, 0, 0 ], textOffset(1, 0));
-        const firstAndLastColTds = new RegExp(`<td>\\d+-(${lastColIndex}|0)</td>`, 'g');
+        const firstAndLastColTds = new RegExp(`<td>\\d+-0</td>`, 'g');
```

Comment:
As I made a not of in the comment of this test-file it was because this test depends on the test before it so it became extremely temperamental. I changed the `before` to `beforeEach` and changed the test to no longer rely on the test before it, solving this problem.